electron/menu.ts

```diff
@@ -92,6 +99,30 @@ const fileMenu = (state: RootState, dispatch: MainDispatch): MenuItemConstructor
         },
       },
       {type: 'separator'},
+      {
+        label: 'New Project',
+        submenu: [
+          {
+            label: 'Select Folder',
+            click: () => {
+              dispatch(openFolderExplorer());
```

Comment:
I think Monokle needs to run these actions in the renderer thread - you'll need to send an event via ipc to trigger this instead of dispatching them to the store directly

Reply:
This dispatch is coming from `main.ts`. And It sends. the action to the current window. 

https://github.com/kubeshop/monokle/blob/main/electron/main.ts#L308

Reply:
but anyway it should send it to focused window - fixing it

Reply:
ok thanks - my bad! Shouldn't we be using this dispatch when selecting an existing project then?

Reply:
This dispatch is for selecting an existing folder - no matter if it is an existing project or not - If the path is bound with an existing project then it will open it otherwise it will create it.

Reply:
If there is a problem with it I will prefer to fix it alongside fixing the non-existing path issue. Wdyt @olensmar ?